libs/designer/src/lib/core/state/connection/connectionSelector.ts

```diff
@@ -39,6 +39,25 @@ export const useConnectorOnly = (connectorId: string | undefined, enabled = true
   );
 };
 
+export const useConnectorsOnly = (connectorIds?: string[]): UseQueryResult<Connector[] | undefined, unknown> => {
+  return useQuery(
+    ['apiOnly', connectorIds],
```

Comment:
Please do not use same key we use for single connector, use apis and we can do useConnectors if this name is not already used